JCDispatch/src/main/java/lib/nameParsing.java

```diff
@@ -0,0 +1,155 @@
+package lib;
```

Comment:
All files pertaining to an integration should be placed inside the same directory, which in your case would be `JCDispatch/src/main/java/com/openlattice/integrations/jcDispatch`. Two things:

1. This file is in the wrong location, and should be moved somewhere inside `JCDispatch/src/main/java/com/openlattice/integrations/jcDispatch`
2. `package lib;` should be changed to `package com.openlattice.integrations.jcDispatch;`